etc/helm/examples/local-dev-values.yaml

```diff
@@ -15,11 +15,21 @@ pachd:
     enabled: false
   clusterDeploymentID: dev
   enterpriseLicenseKey: ""
+  pachOAuth:
+    clientId: "pachd"
+    issuer: "http://pachd:1658/"
+    redirectURI: "http://localhost:30657/authorization-code/callback"
+  upstreamIDP:
+    mockIDP: true
+    issuer: ""
+    clientID: ""
+    clientSecret: ""
+    redirectURI: "http://localhost:30658/callback"
 
 console:
   enabled: false
   config:
-    issuerURI: "http://pachd:1658"
+    issuerURI: "http://pachd:1658/"
```

Comment:
Is a trailing slash always required here?

Reply:
verified that the backslash isn't essential. The issuer URI value just needs to be consistent between console and pachd